app.go

```diff
@@ -256,6 +256,23 @@ func (l loggerOption) String() string {
 	return fmt.Sprintf("fx.Logger(%v)", l.p)
 }
 
+func DisableSignalHandler() Option {
```

Comment:
If it would be used only for test purpose, I would expect it to have a *testing.T as argument.

This would prevent anyone to use it for non test purpose

Reply:
It's generally useful. I want to use it for normal operation (not tests). See attached issue.